arrow-continuations/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id "maven-publish"
+    id "base"
+    id "org.jetbrains.kotlin.jvm"
+    id "org.jetbrains.kotlin.kapt"
+    id "org.jetbrains.dokka"
+    id "org.jlleitschuh.gradle.ktlint"
+    id "ru.vyarus.animalsniffer"
+}
+
+apply from: "$SUB_PROJECT"
+apply from: "$DOC_CREATION"
+apply plugin: 'kotlinx-atomicfu'
+
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
+    compile project(":arrow-annotations")
+    compile project(":arrow-core")
```

Comment:
`compileOnly` ?

Reply:
I am pretty sure most if not all deps for this module can actually be dropped, will check this tomorrow :+1:

Reply:
This code actually should live inside core because all the Fx blocks there need to be refactored. We initially moved it to its own module to avoid compiling all of core while developing. Unless the fx blocks are reachable from this module we should probably move it to core. @1Jajen1 as you mentioned we need to explore handlers vs instances and related findings before 1.0.

Reply:
We can offer continuations as a single module and expose fx constructors from core by adding this as a dep right? 
Continuations have 0 dependencies on core atm, so anyone looking to use them without all the stuff from core can do so.

Not sure what the right call here is, I can copy stuff over either way :+1:

Reply:
If that is the case then perfect I thought we depended in core for some reason. Awesome!